src/CSModule.sol

```diff
@@ -880,7 +880,7 @@ contract CSModule is
     /// @notice Called by Staking Router
     /// @param nodeOperatorIds IDs of the Node Operators to decrease totalVettedKeys for
     /// @param vettedKeysByOperator Corresponding values for totalVettedKeys decrease
-    function decreaseOperatorVettedKeys(
+    function decreaseVettedSigningKeysCount(
         uint256[] calldata nodeOperatorIds,
         uint256[] calldata vettedKeysByOperator
```

Comment:
these args are bytes packed probably same way as in the stuck and exited
```
    /// @notice Called by StakingRouter to decrease the number of vetted keys for node operator with given id
    /// @param _nodeOperatorIds bytes packed array of the node operators id
    /// @param _vettedSigningKeysCounts bytes packed array of the new number of vetted keys for the node operators
    function decreaseVettedSigningKeysCount(
        bytes calldata _nodeOperatorIds,
        bytes calldata _vettedSigningKeysCounts
    ) external;

```

Reply:
Done